### ISSUE TITLE
Update README

### ISSUE THREAD

**Issue by Demfier** [Demfier/GitPub#6]
The current README seems a bit disorganized.

**Comment by apsknight** [comment]
Hey! Please elaborate on what is expected from README ?

**Comment by Demfier** [comment]
@apsknight Its good to see your enthusiasm to contribute but kindly DO NOT start coding before Dec 1. However, you are free to discuss anything related to your future work.
Please have a look at the [timeline](https://kwoc.kossiitkgp.in/#timeline) for any confusion.

Coming to your PR #7:
- The changes look alright but I am closing your PR for now as your commits won't be count by kwoc.
- Also, have a look at [CONTRIBUTION.md](https://github.com/demfier/gitpub/tree/master/CONTRIBUTING.md) before submitting a PR. I see you haven't created a seperate branch for the readme update. While it's okay for these smaller PRs, it is not recommended when you handle a more convoluted task. So it better to follow the code of conduct from the start itself. After all, we are here to learn! :smile: 

Cheers!

**Comment by apsknight** [comment]
@Demfier Thanks for your quick reply. I'll follow contribution guidelines in future.

**Comment by monsij** [comment]
Hello there ,i want to contribute for this project. Please get me started

**Comment by Demfier** [comment]
Hey @monsij 
The formatting and structure of the current README isn't so great. You can go through md file and you'll realize what I mean. Also, have a look at #7 by @apsknight to get an idea. It would be great if you could come with some more updates in addition to what is present in the PR. However, please don't copy what is already present there. 

I am adding you both as a collabortor. Please self-assign the issue and start working on it.

All the best!

**Comment by vchrombie** [comment]
Can I help you with this issue?
Can we share the work?
Is this assigned to only one member?

I am interested in contributing to this project.

**Comment by monsij** [comment]
I've given it a try but i don't know why the build tests are failing.Maybe it has nothing to do with README.md

**Comment by apsknight** [comment]
Hey @Demfier 

Please look at https://github.com/apsknight/GitPub/blob/master/README.md and let me know if any other changes are required or else may I create a PR now ?

**Comment by Demfier** [comment]
Hey @apsknight - Please create a PR. I can add my review then, and btw gitpub can be used via `pip install gitpub`, so you might wanna update that in your README and add something about Python in the Prerequites.

Also, make sure that you go through [CONTRIBUTING.md](https://github.com/demfier/gitpub/tree/master/CONTRIBUTING.md) before making a PR.
